### DISCUSSION TITLE
Welcome to introductions Discussions!

### DISCUSSION THREAD

**Discussion by JLambertazzo** [WebbieHub/introductions#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by judy-n** [comment]
Hi! I'm Judy, I'm a 3rd year CS student and exec at UofT Web. I haven't done much open-source before so I'm really excited to get into it through this community 😸

**Reply by JLambertazzo** [reply]
Hi Judy!!
Welcome to WebbieHub 😄
